Fix linting issue preventing us from updating to flake8 4.7.1

https://github.com/mozilla/bugbug/pull/135 is failing because:
> ./bugbug/model.py:102:21: E117 over-indented

We should fix it, and then we can merge https://github.com/mozilla/bugbug/pull/135.

--- a/bugbug/model.py
+++ b/bugbug/model.py
@@ -99,7 +99,7 @@ class Model():
 
             print(f'\nTop {len(important_features)} Features :')
             for i, [importance, index, is_positive] in enumerate(important_features):
-                    print(f'{i + 1}. \'{feature_names[int(index)]}\' ({"+" if (is_positive) else "-"}{importance})')
+                print(f'{i + 1}. \'{feature_names[int(index)]}\' ({"+" if (is_positive) else "-"}{importance})')
 
         y_pred = self.clf.predict(X_test)
 

